lib/foreman_discovery/engine.rb

```diff
@@ -45,16 +45,23 @@ class Engine < ::Rails::Engine
             :discovered_hosts          => [:index, :show, :auto_complete_search],
             :"api/v2/discovered_hosts" => [:index, :show]
           }, :resource_type => 'Host'
-          permission :provision_discovered_hosts, {
+          permission :submit_discovered_hosts, {
+            :"api/v2/discovered_hosts" => [:facts, :create]
+          }, :resource_type => 'Host'
+          permission :auto_provision_discovered_hosts, {
             :discovered_hosts          => [:auto_provision, :auto_provision_all],
-            :"api/v2/discovered_hosts" => [:create, :auto_provision, :auto_provision_all]
```

Comment:
FYI `create` action was removed by https://github.com/theforeman/foreman_discovery/commit/1f30d55ff87cb2a04b421a2b119275d40e268411 but we forgot to remove the permissions here